tests/components/plex/test_config_flow.py

```diff
@@ -334,47 +342,44 @@ def init_config_flow(hass):
-            ],
-            config_flow.CONF_SERVER: MOCK_SERVERS[1][config_flow.CONF_SERVER],
+            config_flow.CONF_SERVER_IDENTIFIER:
+            MOCK_SERVERS[1][config_flow.CONF_SERVER_IDENTIFIER],
+            config_flow.CONF_SERVER:
+            MOCK_SERVERS[1][config_flow.CONF_SERVER],
         },
     ).add_to_hass(hass)
 
     result = await hass.config_entries.flow.async_init(
-        config_flow.DOMAIN, context={"source": "user"}
-    )
+        config_flow.DOMAIN, context={"source": "user"})
     assert result["type"] == "form"
     assert result["step_id"] == "start_website_auth"
 
-    with patch(
-        "plexapi.myplex.MyPlexAccount", return_value=MockPlexAccount(servers=2)
-    ), patch(
-        "plexapi.server.PlexServer", return_value=mock_plex_server
-    ), asynctest.patch(
-        "plexauth.PlexAuth.initiate_auth"
-    ), asynctest.patch(
-        "plexauth.PlexAuth.token", return_value=MOCK_TOKEN
-    ):
-        result = await hass.config_entries.flow.async_configure(result["flow_id"])
+    with patch("plexapi.myplex.MyPlexAccount",
+               return_value=MockPlexAccount(servers=2)), patch(
+                   "plexapi.server.PlexServer",
+                   return_value=mock_plex_server), asynctest.patch(
+                       "plexauth.PlexAuth.initiate_auth"), asynctest.patch(
+                           "plexauth.PlexAuth.token", return_value=MOCK_TOKEN):
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"])
         assert result["type"] == "external"
 
-        result = await hass.config_entries.flow.async_configure(result["flow_id"])
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"])
         assert result["type"] == "external_done"
 
-        result = await hass.config_entries.flow.async_configure(result["flow_id"])
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"])
         assert result["type"] == "create_entry"
         assert result["title"] == mock_plex_server.friendlyName
-        assert result["data"][config_flow.CONF_SERVER] == mock_plex_server.friendlyName
-        assert (
-            result["data"][config_flow.CONF_SERVER_IDENTIFIER]
-            == mock_plex_server.machineIdentifier
-        )
-        assert (
-            result["data"][config_flow.PLEX_SERVER_CONFIG][CONF_URL]
-            == mock_plex_server.url_in_use
-        )
-        assert result["data"][config_flow.PLEX_SERVER_CONFIG][CONF_TOKEN] == MOCK_TOKEN
+        assert result["data"][
+            config_flow.CONF_SERVER] == mock_plex_server.friendlyName
+        assert (result["data"][config_flow.CONF_SERVER_IDENTIFIER] ==
+                mock_plex_server.machineIdentifier)
+        assert (result["data"][config_flow.PLEX_SERVER_CONFIG][CONF_URL] ==
+                mock_plex_server.url_in_use)
+        assert result["data"][
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636425)